app/client/src/components/JoinButton.jsx

```diff
@@ -48,7 +48,12 @@ const JoinButton = (props) => {
   return (
     <>
       {isInCommunity && (
-        <IconButton variant="contained" color={"error"} onClick={handleLeave}>
+        <IconButton
+          variant="contained"
+          disableRipple
```

Comment:
Would we be able to keep the ripple and solve the oversized button issue at a higher level in css?